redis/map.jinja

```diff
@@ -7,12 +7,14 @@
 	  'wheezy': '2.4',
 	  'jessie': '2.8',
 	  'default': '2.8'
-	}, grain='oscodename'),
+	  }, grain='oscodename'),
+	'logfile': '/var/log/redis/redis-server.log',
```

Comment:
Indent this to match keys above. Also this should go above the line ``}, grain='oscodename'),``